src/node/crypto.ts

```diff
@@ -136,26 +116,60 @@ export {
   Certificate,
   // X509
   X509Certificate,
-}
+};
 
 export function getCiphers() {
-  return ["aes-128-cbc", "aes-192-cbc", "aes-256-cbc", "aes-128-ctr", "aes-192-ctr", "aes-256-ctr",
-  "aes-128-ecb", "aes-192-ecb", "aes-256-ecb", "aes-128-gcm", "aes-192-gcm", "aes-256-gcm",
-  "aes-128-ofb", "aes-192-ofb", "aes-256-ofb", "des-ecb", "des-ede", "des-ede-cbc", "rc2-cbc"];
+  return [
```

Comment:
Ditto: I think this needs to be ignored as well.

Reply:
I personally feel that benefits of saving time and having consistent formatting outweigh choosing a good style for each block of code. I would only ignore extremely egregious formatting changes.

Reply:
I agree with @npaun but do we want to change the config so that arrays are more concise in general?
That way we don't need to ignore this.